src/gh_pr/plugins/loader.py

```
@@ -0,0 +1,301 @@
+            path: Plugin directory path
+
+        Returns:
+            Plugin instance or None
+        """
+        # Look for main plugin module
+        main_file = path / 'plugin.py'
+        if not main_file.exists():
+            main_file = path / '__init__.py'
+
+        if not main_file.exists():
+            raise FileNotFoundError(f"No plugin.py or __init__.py found in {path}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)